common/src/main/java/org/corfudb/common/metrics/micrometer/MicroMeterUtils.java

```diff
@@ -191,6 +193,10 @@ public static void counterIncrement(double value, String name, String... tags) {
         filterGetInstance(name).ifPresent(registry -> registry.counter(name, tags).increment(value));
     }
 
+    public static Optional<FunctionCounter> functionCounter(String name, String... tags) {
```

Comment:
The purpose of the function counters is to track a monotonically increasing function of some state. 
If you want to provide the APIs for this I would provide two public methods like that:
```
public static <T extends Number> Optional<FunctionCounter> functionCounter(String name, T state, String... tags) {
        return filterGetInstance(name).map(registry -> registry.more().counter(name, Tags.of(tags), state));
    }
```
and
```
public static <T extends Number> Optional<FunctionCounter> functionCounter(String name, T state,
                                                                               ToDoubleFunction<T> stateFunction, String... tags) {
        return filterGetInstance(name).map(registry -> FunctionCounter.builder(name, state, stateFunction).tags(tags).register(registry));
    }
```

Reply:
Moving the table.count metric to UfoCompactor in nsx. Removed the functionCounter as it's no longer needed